.travis.yml

```diff
@@ -17,6 +17,8 @@ matrix:
       env: TOXENV=py36
     - python: 3.7
       env: TOXENV=py37
+    - python: 3.8-dev
```

Comment:
Given that 3.8 has been released now, can we update this yet on Travis?

Reply:
It's included in [this list](https://docs.travis-ci.com/user/languages/python/#specifying-python-versions), so let's try it out.

Reply:
It's included in [this list](https://docs.travis-ci.com/user/languages/python/#specifying-python-versions), so let's try it out.